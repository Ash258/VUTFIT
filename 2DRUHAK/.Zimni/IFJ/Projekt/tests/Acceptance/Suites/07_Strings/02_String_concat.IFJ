' String concat

scope
	dim a as string
	dim content as string = !" my content "

	print a; content; !"\n";

	dim b as string = !""
	dim c as string = !"foo bar"
	dim foo as string
	foo = c + !"" + c + !"" + c + !"\n"

	print b; foo; foo; content;
end scope
