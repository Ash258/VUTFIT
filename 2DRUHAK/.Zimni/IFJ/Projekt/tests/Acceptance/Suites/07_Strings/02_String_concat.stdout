 my content 
foo barfoo barfoo bar
foo barfoo barfoo bar
 my content 